content/en/docs/Tasks/update-and-ship-operator.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Update and ship an Operator"
+date: 2021-03-10
+weight: 2
+description: > 
+    In depth explanation of operator update and shipment process. 
+--- 
+
+## Introduction
```

Comment:
I think Overview follows more the standard.

Reply:
Agree. Updated as per suggestion.